db/dbHelper/model.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     //             `actions.flag as actions_flag`)
     //     .innerJoin(`projects`, `projects.id`, `actions.id`)
     //     .where(`projects.id`, id)
-    const project = await db(`actions`).where({
```

Comment:
Good catch here. Always remember to check that everything matches up 😉
Also, can remove unused commented out code above. 👍